bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java

```diff
@@ -1030,7 +1030,7 @@ public void removeSpectator(@NotNull Player p, boolean disconnect) {
         /* Remove also the party */
         if (getParty().hasParty(p)) {
             if (getParty().isOwner(p)) {
-                if (status != GameState.restarting) {
+                if (status != GameState.restarting || status != GameState.playing) {
```

Comment:
what is it going to do with party members if the owner joins another arena right after leaving? while the party members are still alive?